.jenkins/build_docker_images.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Set variable to allow parallel building in the Docker image creation
+export DOCKER_BUILDKIT=1
+
+# Get the dockerfiles for the images to be tested on Piz Daint
+dockerfiles=( $( ls docker/Dockerfile.gnu* ) )
+
+for df in ${dockerfiles[@]}; do
+
+   # Grab the architecture tag from the dockerfile filename
+   arch=${df:18}
+   container=us.gcr.io/vcm-ml/fv3gfs-compiled:$arch
+   tar_file=fv3gfs-compiled_${arch}.tar
+   
+   echo " "
+   echo "Building fv3gfs-compiled:${arch}"
+   echo " "
+   
+   # Build the docker image and push image to VCM's Google Container Repository
+   docker build -f $df --target fv3-bld -t $container .
+   docker push $container
+
+   # Copy an archived version of the image to a public Google Storage Bucket
+   docker save $container -o $tar_file
```

Comment:
Could this be gzipped to save cloud space?
`docker save $container | gzip > ${tar_file}.gz`

Reply:
I guess it could.  It would be an extra couple of steps to the CI plan (packing and then extracting the tar archive).  I'll test it and see what happens

Reply:
Cool!

Reply:
Yup.  It works fine.  I have slipped in the Intel-compiled Docker image as well and I am running the Jenkins tests now.

Reply:
Nice work, that should save a little scratch.